opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_view_instrument_match.py

```diff
@@ -68,7 +69,7 @@ def consume_measurement(self, measurement: Measurement) -> None:
 
         if attributes not in self._attributes_aggregation.keys():
             with self._lock:
-                self._attributes_aggregation[attributes] = self._aggregation()
+                self._attributes_aggregation[attributes] = self._aggregation
```

Comment:
This is causing the same instance of `Aggregation` to be used across all of the attributes sets. We need a distinct aggregation per attributeset

Reply:
SHould be fixed in #2482, just need to rebase

Reply:
Right, I have removed aggregation from the view arguments for the time being.